lib/utils.pm

```diff
@@ -1057,6 +1058,24 @@ sub is_boot_encrypted {
     return 1;
 }
 
+=head2 need_passphrase_again
+
+ need_passphrase_again();
+
+With newer grub2 (in TW and SLE15-SP6 currently), if root disk is encrypted and
+contains `/boot`, entering the passphrase in GRUB2 is enough. The key is passed
+on during boot, so it's not asked for a second time.
+We need to enter the passphrase again if there are separate partitions encrypted
+without LVM configuration (cr_swap,cr_home etc).
+
+=cut
+
+sub need_passphrase_again {
+    my $need_passphrase_again = is_leap || is_sle('<15-sp6') || is_leap_micro || is_sle_micro || is_alp || (!get_var('LVM', '0') && !get_var('FULL_LVM_ENCRYPT', '0'));
```

Comment:
Leap has same packages shared with SLE15 thus the same issue does appear on 15.6 testresults also, see https://openqa.opensuse.org/tests/3804783 , we should follow sle's condition to use `is_leap('<15.6')` here

Reply:
Thanks, I can see https://openqa.opensuse.org/tests/3804783#step/boot_encrypt/6
Let me do that

Reply:
https://openqa.opensuse.org/tests/3805337# passed